.travis.yml

```diff
@@ -23,7 +24,32 @@ matrix:
     - rust: nightly
       env: FEATURES=""
   allow_failures:
-    - rust: nightly
+  - rust: nightly
 
 script:
   - cargo test --no-default-features --features "$FEATURES"
+
+before_deploy:
+- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    make dmg;
+    export DEPLOY_FILE="$PWD/target/release/osx/Alacritty.dmg";
+  fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    cargo build --release;
+    cargo install cargo-deb;
+    export DEPLOY_FILE=$( cargo deb --no-build );
+  fi
+- echo "DEPLOYING FILE $DEPLOY_FILE";
+
+deploy:
+  provider: releases
+  api_key:
+    secure: o2Q39A2JAKeMEYD0dFJTpYJMpAsHYcUa4DIXApqn/ygXQzcfPpnLelCWb70zOlICIk3ICETlzeLfF7s/q802jy5aEFLbbmIp+rT0SWlRiShuQqLPkwlK7Cqh2nUAVPoozYn1y9ha8kozOL3xgDxI9RGHMC+7ZV9duVLWUqz5Ah7qBoOx9QemDMrdPJ7zi8vFvJp4oW3EV7wQRRfEmFkqOAxbAWhLeTNGVR9ECjUZIgXWRz1kjqVz6dJrLauH/D068RSV/7JM1dlzrMkBsmRe8xVOTnAiHIoGMB4UaUTLz8hJNH+KUmXfprfBKuFEQ7jDnYq+6/y200VYuRfRsU2EOqYjzHXmWD6E3vkHhVdrUUfFmG9tfAzIXFZxs0LVIszWgqqMypmuLNxgYcfpMmo+YeHGWX59HOVwXmvY38ix1++zLh44kpRuX0mYFuPdH6gRdeKPr7jyueODKRwsegh1d8rVl7T0HQpBmDcWHBidOf8Aw8NTrsEoRQFlS/eVZyGnZet/nf6Otenp/5MIBA3BHO9R9QaCo0FT3Yi7FL/kkoEZs+lN5mKJulZ6hpmG760h+mYBqw7buLKcCR3qBiuXOpgfzNWgOoAGb3G7eUAhCf1zHTEcdL6U/SEA2ofpaPZGqrcuZ7aTgnJ6lWU4YWI8X+hAouv6IOk/UoxiONIUwHE=
+  skip_cleanup: true
+  overwrite: true
+  file_glob: true
+  file: $DEPLOY_FILE
```

Comment:
Since you're already working on this, it might make sense to also include uploading the binary directly. So this PR would just do everything we need to deploy binaries to github.

You can upload multiple files like this:
```
file:
    - "FILE 1"
    - "FILE 2"
```

Since you have a `cargo build --release` in the `before_deploy`, you could just upload that file directly. Regarding naming, I'd propose something like this: `alacritty-0.18.3` and `alacritty-0.18.3.deb`. It should be possible to do this using the tag name, so just `alacritty-TAG` for the binary and `alacritty-TAG.deb` for the `.deb` file.

Reply:
Uploading the binary is a good idea. I suggest we do `alacritty-TAG-ARCH`.